packages/mobile/src/backup/__snapshots__/BackupPhraseContainer.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`renders correctly for input backup phrase 1`] = `
+exports[` renders correctly for input backup phrase 1`] = `
```

Comment:
remove ?

Reply:
This was a weird error I was seeing with the mobile tests. I'll try removing it but there's a chance the mobile tests fail again. I chatted with Rossy about it and he said it was weird but probably not a big deal. Beyond me why jest would change this for just one snapshot

Reply:
Could you add a comment there? To avoid someone changing it in the future by mistake